include/net/packet.h

```diff
@@ -164,14 +165,14 @@ namespace ace { namespace net {
         ChangeWeapon,
     };
 
-    enum class WEAPON : uint8_t {
+    enum class WEAPON : int8_t {
```

Comment:
These types should be uint8_t. If the negative overflow is causing a stink within GCC, it would be better to just replace the -1s with 255 or `std::numeric_limits<uint8_t>::max()` if you're feeling verbose